README.md

```diff
@@ -5,7 +5,7 @@ This is a work-in-progress. It's alpha quality. Feedback appreciated.
 # Why?
 
 The Ember addons ecosystem is one of the best things about Ember. But
-users stuck on legacy build systems have a hard time taking advantage
+users of non ember-cli build systems have a hard time taking advantage
```

Comment:
I don't know if there has been an "official" word on that, but from what I've seen it definitely won't be a priority going forward. Idiomatic, no; ember-cli is the prescribed way going forward. Incorrect? I think too strong of word to know at this point.